### PR TITLE
Simplified

### PR DESCRIPTION
**Pull request by blanquer**

### CODE REVIEW

File: lib/praxis-blueprints/blueprint.rb [hunk 1]
```diff
@@ -36,37 +55,33 @@ def self.inherited(klass)
...
-      blueprint.send(:initialize, object, decorators)
+      blueprint.send(:initialize, object)
       blueprint
     end
 
     def self.family
       'hash'
     end
 
+    # TODO: should we get rid of shallow? (or describe in general?)
```
Comment (careo) [inline lib/praxis-blueprints/blueprint.rb]: I think we use describe it in the docs. Not sure if we need shallow anymore though or not, good question

File: lib/praxis-blueprints/blueprint.rb [hunk 2]
```diff
@@ -114,20 +123,15 @@ def self.domain_model(klass = nil)
...
-      end
+      Attributor::Struct.check_option!(name, value)
     end
 
     def self.load(value, context = Attributor::DEFAULT_ROOT_CONTEXT, **options)
       case value
       when self
         value
       when nil, Hash, String
+        # TODO: huh? what if loading fails...shouldn't we capture errors? probably no, in validate?
```
Comment (careo) [inline lib/praxis-blueprints/blueprint.rb]: Loading can't silently fail, it either works or throws an exception.

File: lib/praxis-blueprints/blueprint.rb (hunk 1 above)
Comment (blanquer) [reply lib/praxis-blueprints/blueprint.rb]: we "used" to use them in the Praxis docs...but now we moved to OpenAPI docs and we technically do not "need" describe anymore :)
I was pseudo-hesitant to remove it all...but maybe we just do :)
Attributor will still have all the describes, but...we can remove that later.

Comment (careo) [reply lib/praxis-blueprints/blueprint.rb]: Then let's just remove it.

**Comment by blanquer** [comment]
The core of this gem has been simplified and moved into the praxis main repo

**Pull request closed without merging**